packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -780,8 +790,9 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
   @autobind
   private _onRenderList(props: IComboBoxProps): JSX.Element {
     let {
-      onRenderItem = this._onRenderItem
```

Comment:
No need for these default values as they were already resolved in the main render function